features/orabos/info.yaml

```diff
@@ -0,0 +1,7 @@
+description: "ORA Baremetal OS"
```

Comment:
this combination is difficult. the gardener feature has some bias here that could be avoided by the chost feature.
the other is the combination vhost + chost is that really intended?